rhizome/host/lib/vm_setup.rb

```diff
@@ -142,6 +142,12 @@ def purge_storage
     rm_if_exists(vp.storage_root)
   end
 
+  def unmount_hugepages
+    r "umount #{vp.q_hugepages}" if File.exist?(vp.hugepages)
```

Comment:
```suggestion
    r "umount #{vp.q_hugepages}"
```

Given you handle the exception anyway (which is good), why bother checking first? Rough justification: same as why "file exist? then delete" is less good than checking for ENOENT after trying.

Reply:
I think it also lets you cut some tests

Reply:
I checked it since I got a different exception (File not exists) in my local setup when the directory didn't exist. Sure, I can add it to the regex.